open_prices/prices/models.py

```diff
@@ -133,6 +134,13 @@ class Price(models.Model):
         related_name="prices",
     )
 
+    receipt_quantity = models.PositiveBigIntegerField(
```

Comment:
I wouldn't expect this value to reach BigInteger levels.
`PositiveSmallIntegerField` is probably enough (<32767)

Reply:
nice catch ! I think I went a bit fast and copy/pasted the setup of the `osm_id` field. will change to `PositiveIntegerField`